decidim-comments/app/mailers/decidim/comments/comment_notification_mailer.rb

```diff
@@ -26,6 +28,12 @@ def reply_created(user, reply, comment, commentable)
           mail(to: comment.author.email, subject: subject)
         end
       end
+
+      private
+
+      def commentable_title
+        @commentable.title.kind_of?(Hash) ? @commentable.title[I18n.locale.to_s] : @commentable.title
```

Comment:
We should pinpoint this later, with a refactor for the notifications, as we talked offline with @oriolgual and @beagleknight